### PR TITLE
OCP4.3 Welcome page OSDOCS-767

### PR DESCRIPTION
**Pull request by chrisnegus**
Incorporated updates from OpenShift writers for 4.3.

### CODE REVIEW

File: welcome/index.adoc [hunk 1]
```diff
@@ -52,22 +52,36 @@ xref:../installing/installing_openstack/installing-openstack-troubleshooting.ado
 
 - **Install a cluster in a restricted network**: If your cluster that uses
 user-provisioned infrastructure on
-xref:../installing/installing_aws/installing-restricted-networks-aws.adoc#installing-restricted-networks-aws[AWS],
+xref:../installing/installing_aws/installing-restricted-networks-aws.adoc[AWS],
```
Comment (vikram-redhat) [inline welcome/index.adoc]: needs an anchor id.

File: welcome/index.adoc [hunk 2]
```diff
@@ -52,22 +52,36 @@ xref:../installing/installing_openstack/installing-openstack-troubleshooting.ado
...
 GCP,
-xref:../installing/installing_vsphere/installing-restricted-networks-vsphere.adoc#installing-restricted-networks-vsphere[vSphere],
+xref:../installing/installing_vsphere/installing-restricted-networks-vsphere.adoc[vSphere],
```
Comment (vikram-redhat) [inline welcome/index.adoc]: Same, an anchor id is required.

File: welcome/index.adoc [hunk 3]
```diff
@@ -52,22 +52,36 @@ xref:../installing/installing_openstack/installing-openstack-troubleshooting.ado
...
 or
-xref:../installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc#installing-restricted-networks-bare-metal[bare metal]
+xref:../installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc[bare metal]
```
Comment (vikram-redhat) [inline welcome/index.adoc]: Same.

File: welcome/index.adoc [hunk 4]
```diff
@@ -52,22 +52,36 @@ xref:../installing/installing_openstack/installing-openstack-troubleshooting.ado
...
 does not have full access to the internet, you can
 xref:../installing/install_config/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[mirror the {product-title} installation images]
 and install a cluster in a restricted network.
 
+- **Install a cluster in an existing network**: If you use an existing VPC in
+xref:../installing/installing_aws/installing-aws-vpc.adoc[AWS] or
```
Comment (vikram-redhat) [inline welcome/index.adoc]: Anchor ids in all of these.

File: welcome/index.adoc [hunk 5]
```diff
@@ -52,22 +52,36 @@ xref:../installing/installing_openstack/installing-openstack-troubleshooting.ado
...
+xref:../installing/installing_gcp/installing-gcp-vpc.adoc[GCP] or an existing
+xref:../installing/installing_azure/installing-azure-vnet.adoc[VNet]
+on Azure, you can install a cluster 
+
+- **Install a private cluster**: If your cluster does not require external
+internet access, you can install a private cluster on
+xref:../installing/installing_aws/installing-aws-private.html[AWS],
```
Comment (vikram-redhat) [inline welcome/index.adoc]: Not html, but .adoc, with anchor ids.

File: welcome/index.adoc [hunk 6]
```diff
@@ -88,11 +102,17 @@ The odo CLI tool lets developers create single or multi-component applications
 easily and automates deployment, build, and service route configurations.
 It abstracts complex Kubernetes and {product-title} concepts, allowing developers to focus on developing their applications.
 
+- **Deploy Helm Charts**: 
+xref:../cli_reference/helm-cli/getting-started-with-helm-on-openshift-container-platform.html[Helm 3]
```
Comment (vikram-redhat) [inline welcome/index.adoc]: Not html, but adoc with anchor ids.

File: welcome/index.adoc (hunk 1 above)
Comment (chrisnegus) [reply welcome/index.adoc]: Thanks. Done.

File: welcome/index.adoc (hunk 2 above)
Comment (chrisnegus) [reply welcome/index.adoc]: Done.

File: welcome/index.adoc (hunk 3 above)
Comment (chrisnegus) [reply welcome/index.adoc]: Done.

File: welcome/index.adoc (hunk 4 above)
Comment (chrisnegus) [reply welcome/index.adoc]: Done.

File: welcome/index.adoc (hunk 5 above)
Comment (chrisnegus) [reply welcome/index.adoc]: Done.

File: welcome/index.adoc (hunk 6 above)
Comment (chrisnegus) [reply welcome/index.adoc]: Done.

File: welcome/index.adoc [hunk 7]
```diff
@@ -61,13 +61,27 @@ does not have full access to the internet, you can
 xref:../installing/install_config/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[mirror the {product-title} installation images]
 and install a cluster in a restricted network.
 
-- **xref:../installing/installing-gather-logs.adoc#[Check installation logs]**: Access installation logs to evaluate issues that occur during {product-title} {product-version} installation.
+- **Install a cluster in an existing network**: If you use an existing VPC in
```
Comment (jboxman) [inline welcome/index.adoc]: Do we want _in_ or _on_ here? Elsewhere we're saying _install on..._

Comment (jboxman) [inline welcome/index.adoc]: Do we want to define VPC? I had to look, hilariously, but it's _Virtual Private Cloud_.

File: welcome/index.adoc [hunk 8]
```diff
@@ -61,13 +61,27 @@ does not have full access to the internet, you can
...
+xref:../installing/installing_aws/installing-aws-vpc.adoc#installing-aws-vpc[AWS] or
+xref:../installing/installing_gcp/installing-gcp-vpc.adoc#installing-gcp-vpc[GCP] or an existing
+xref:../installing/installing_azure/installing-azure-vnet.adoc#installing-azure-vnet[VNet]
+on Azure, you can install a cluster
```
Comment (jboxman) [inline welcome/index.adoc]: Is the end of the sentence missing after _cluster_?

File: welcome/index.adoc [hunk 9]
```diff
@@ -61,13 +61,27 @@ does not have full access to the internet, you can
...
+xref:../installing/installing_gcp/installing-gcp-vpc.adoc#installing-gcp-vpc[GCP] or an existing
+xref:../installing/installing_azure/installing-azure-vnet.adoc#installing-azure-vnet[VNet]
+on Azure, you can install a cluster 
+
+- **Install a private cluster**: If your cluster does not require external
+internet access, you can install a private cluster on
+xref:../installing/installing_aws/installing-aws-private.adoc#installing-aws-private[AWS],
+xref:../installing/installing_azure/installing-azure-private.adoc#installing-aws-private[Azure], or
+xref:../installing/installing_gcp/installing-gcp-private.adoc#installing-gcp-private[GCP].
+You still require internet access to access the cloud APIs and installation
```
Comment (jboxman) [inline welcome/index.adoc]: Or

s/You still require internet access to access the cloud APIs and installation media./Internet access is still required to access the cloud APIs and installation media./

File: welcome/index.adoc [hunk 10]
```diff
@@ -88,11 +102,17 @@ The odo CLI tool lets developers create single or multi-component applications
...
 
-- **xref:../openshift_images/images-understand.adoc#understanding-images[Create container images]**: A container image is the most basic building block in OpenShift (and Kubernetes) applications. Defining imagestreams lets you gather multiple versions of an image in one place as you continue its development. S2I containers let you insert your source code into a base container that is set up to run code of a particular type (such as Ruby, Node.js, or Python).
+- **xref:../openshift_images/images-understand.adoc#understanding-images[Create container images]**: A container image is the most basic building block in {product-title} (and Kubernetes) applications. Defining imagestreams lets you gather multiple versions of an image in one place as you continue its development. S2I containers let you insert your source code into a base container that is set up to run code of a particular type (such as Ruby, Node.js, or Python).
```
Comment (jboxman) [inline welcome/index.adoc]: S2I - source to image?

File: welcome/index.adoc [hunk 11]
```diff
@@ -103,7 +123,9 @@ ifdef::openshift-enterprise,openshift-origin[]
 - **xref:../operators/operator_sdk/osdk-getting-started.adoc#osdk-getting-started[Create Operators]**: Operators are the preferred method for creating on-cluster applications for {product-title} {product-version}. Learn the workflow for building, testing, and deploying Operators. Then create your own Operators based on xref:../operators/operator_sdk/osdk-ansible.adoc#osdk-ansible[Ansible] or
 xref:../operators/operator_sdk/osdk-helm.adoc#osdk-helm[Helm], or configure xref:../operators/operator_sdk/osdk-monitoring-prometheus.adoc#osdk-monitoring-prometheus[built-in Prometheus monitoring] using the Operator SDK.
 
-- **Use xref:../applications/service_brokers/installing-template-service-broker.adoc#sb-about-template-service-broker-sb-installing-tsb[Template Service Broker] or xref:../applications/service_brokers/installing-ansible-service-broker.html#sb-about-ansible-service-broker-sb-installing-asb[OpenShift Ansible Broker] applications**: Service brokers are a mechanism for provisioning applications outside of an {product-title} environment.
+- **REST API Guide**: Lists {product-title} application programming interface endpoints.
```
Comment (jboxman) [inline welcome/index.adoc]: `xref:../rest_api/index.adoc#rest-api`

File: welcome/index.adoc [hunk 12]
```diff
@@ -116,18 +138,18 @@ that watch over the cluster. As a cluster administrator, this
...
 by xref:../machine_management/deploying-machine-health-checks.adoc#deploying-machine-health-checks[deploying health checks] and xref:../machine_management/applying-autoscaling.adoc#applying-autoscaling[applying autoscaling to machines].
 
-- **xref:../registry/registry-options.adoc#registry-options[Manage container registries]**: Each {product-title} cluster includes a built-in container registry for storing its images. You can also configure a separate link:https://access.redhat.com/documentation/en-us/red_hat_quay/[Red Hat Quay] registry to use with OpenShift. The link:https://quay.io[Quay.io] site provides a public container registry that stores OpenShift containers and Operators.
+- **xref:../registry/registry-options.adoc#registry-options[Manage container registries]**: Each {product-title} cluster includes a built-in container registry for storing its images. You can also configure a separate link:https://access.redhat.com/documentation/en-us/red_hat_quay/[Red Hat Quay] registry to use with {product-title}. The link:https://quay.io[Quay.io] site provides a public container registry that stores {product-title} containers and Operators.
```
Comment (jboxman) [inline welcome/index.adoc]: s/site/web site/ or is _web_ finally just dropped in 2020?

**Review by jboxman — APPROVED** [review]
Added some comments.

**Comment by jboxman** [comment]
@vikram-redhat, done!

**Comment by vikram-redhat** [comment]
Merging and doing a followup

**Pull request merged**